pkg/commands/workload_create.go

```diff
@@ -68,9 +69,23 @@ func (opts *WorkloadCreateOptions) Exec(ctx context.Context, c *cli.Config) erro
 		workload.Namespace = opts.Namespace
 	}
 
-	if err := c.Get(ctx, client.ObjectKey{Namespace: workload.Namespace, Name: workload.Name}, &cartov1alpha1.Workload{}); !apierrs.IsNotFound(err) {
-		c.Printf("%s workload %q already exists\n", printer.Serrorf("Error:"), fmt.Sprintf("%s/%s", workload.Namespace, workload.Name))
-		return cli.SilenceError(err)
+	existingWorkload := &cartov1alpha1.Workload{}
+
+	err := c.Get(ctx, client.ObjectKey{Namespace: workload.Namespace, Name: workload.Name}, existingWorkload)
+
+	if err != nil {
+		// return err, except when not found
+		if !apierrs.IsNotFound(err) {
+			return err
+		}
+	}
```

Comment:
Do you think that maybe something like this would work?
```suggestion
         // return err, except when not found
        if err := c.Get(ctx, client.ObjectKey{Namespace: workload.Namespace, Name: workload.Name}, existingWorkload); err != nil && !apierrs.IsNotFound(err) {
		return err
	}
```

Reply:
Good point @warango4. I was checking `err` object for two different `apierrs`, but the second check was not valuable. However, your suggestion is spot on as we are down to a single statement. I will update the PR.